compiler/cgen.nim

```diff
@@ -267,20 +273,25 @@ proc safeLineNm(info: TLineInfo): int =
   result = toLinenumber(info)
   if result < 0: result = 0 # negative numbers are not allowed in #line
 
+proc genPostprocessDir(fields: seq[string]): string =
```

Comment:
`fields` contains always exactly 3 strings.

Reply:
For now, yes. In theory, we could add more postprocessing directives with different number of fields. Don't know whether we'll ever need this or not, but I think it's cheap to have this extensibility, just in case.

Reply:
No, the seq creations are unnecessary bloat.

Reply:
What should I use? Open array parameter? Fixed array with 3 elements? Or just three separate parameters?

Reply:
I've decided to use 3 separate parameters, that probably has the least overhead: https://github.com/nim-lang/Nim/pull/23488/commits/657f520e0067bfca49e01c7abef7866cb34e30f5